test/functional/mempool_limit.py

```diff
@@ -78,6 +78,56 @@ def fill_mempool(self):
         assert_equal(node.getmempoolinfo()['minrelaytxfee'], Decimal('0.00001000'))
         assert_greater_than(node.getmempoolinfo()['mempoolminfee'], Decimal('0.00001000'))
 
+    def test_rbf_carveout_disallowed(self):
+        node = self.nodes[0]
+
+        self.log.info("Check that individually-evaluated transactions in a package don't increase package limits for other subpackage parts")
+
+        # We set chain limits to 2 ancestors, 1 descendant, then try to get a parents-and-child chain of 2 in mempool
+        #
+        # A: Solo transaction to be RBF'd (to bump descendant limit for package later)
+        # B: First transaction in package, RBFs A by itself under individual evaluation, which would give it +1 descendant limit
+        # C: Second transaction in package, spends C. If the +1 descendant limit persisted, would make it into mempool
```

Comment:
should say” spend B”.

Reply:
tried pushing an update, it doesn't seem to be showing yet... tried to fix this up anyways!

Reply:
Shows up at ee589d44